ui/button.reel/button.js

```diff
@@ -133,9 +158,8 @@ var Button = exports.Button = Control.specialize(/** @lends module:"montage/ui/n
                 }
             }
 
-            this._label = String(value);
-
-            if (this.isInputElement) {
+            this._label = "" + value;
```

Comment:
please keep what was there: this._label = String(value); it avoids creating a new string if value is already one